pkg/ibmcsidriver/controller_helper.go

```diff
@@ -384,7 +382,7 @@ func overrideParams(logger *zap.Logger, req *csi.CreateVolumeRequest, config *co
 			}
 		case IOPS:
 			// Override IOPS only for custom or dp2
```

Comment:
`Override IOPS only for dp2`

Reply:
Will fix this code comment issue in next PR